src/hotspot/share/gc/shenandoah/shenandoahOldGC.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef SHARE_GC_SHENANDOAH_SHENANDOAHOLDGC_HPP
+#define SHARE_GC_SHENANDOAH_SHENANDOAHOLDGC_HPP
+
+#include "gc/shared/gcCause.hpp"
+#include "gc/shenandoah/shenandoahConcurrentMark.hpp"
+#include "gc/shenandoah/shenandoahConcurrentGC.hpp"
+#include "gc/shenandoah/shenandoahHeap.hpp"
```

Comment:
I see no use of ShenandoahHeap or ShenandoahConcurrentMark in this header. I believe those two includes could be removed.